client/package.json

```diff
@@ -2,33 +2,65 @@
   "name": "sample-items-scoring-guide-client",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "index.tsx",
   "scripts": {
     "less": "lessc-each styles/ dist/css/",
-    "build": "cd src && webpack",
-    "test": "jest -c=test/jest.json"
+    "build": "webpack",
+    "test": "jest",
+    "coverage": "jest --coverage"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {},
   "devDependencies": {
     "@types/enzyme": "2.8.6",
-    "@types/jest": "20.0.8",
+    "@types/jest": "^20.0.8",
```

Comment:
We try to target a specific version instead of latest to ensure everything works the exact same in the future